coverage-report/README.md

```diff
@@ -0,0 +1,16 @@
+## Coverage Report
+
+This module is meant to gather aggregate jacoco test coverage metrics across the `gax-java` and `showcase` modules.
```

Comment:
Would you add why the coverage of the 2 modules, gax-java and showcase, is important?

Reply:
Added more info on the purpose of the metrics being collected.